src/chainlit/__init__.py

```diff
@@ -181,6 +181,15 @@ def sleep(duration: int):
     return asyncio.sleep(duration)
 
 
+__getattr__ = make_module_getattr(
+    {
```

Comment:
Sorry I'm new in python, I wonder why not just import the handlers like this:

```py
from chainlit.langchain.callbacks import LangchainCallbackHandler, AsyncLangchainCallbackHandler, LlamaIndexCallbackHandler, HaystackAgentCallbackHandler
```

Does it has any advantages I didn't know?

Reply:
This would be working fine but would also import the Langchain library by default, which we want to avoid for performance reasons. This PR will make the Langchain (and other libraries) imports lazy.